src/main/java/com/zaxxer/hikari/pool/HikariPool.java

```diff
@@ -631,17 +631,21 @@ public void run()
          previous = now;
 
          logPoolState("Before cleanup\t");
```

Comment:
Go ahead and move the before cleanup and after cleanup log inside of the ``if (idleTimeout < 0)`` conditional.